foreshadow/transformers/externals.py

```diff
@@ -9,7 +9,7 @@
 
 from category_encoders import HashingEncoder, OneHotEncoder
 from sklearn.decomposition import PCA
-from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer, TfidfVectorizer
```

Comment:
Just want to double check with you about adding TfidfTransformer in here? As per Sklearn docs, TfidfVec is CountVec followed by TfidfTrans. The other question is, do we want to add TfidfVectorizer in externals? As of right now, there is a "patched" version that is sitting in internals that is used currently in the smart text transformer.

Reply:
Will remove this and make sure that it all doesn't break.